src/node/node.c

```diff
@@ -324,10 +324,10 @@ struct s_env {
 	char buffer[MAX_ENV_BUF_SZ];
 };
 
-#define MAX_ENV_NB 256
+#define MAX_ENV_NB 64
 struct s_plugin_conf {
-	uid_t uid;
-	gid_t gid;
+	char user[128];
```

Comment:
Can we leverage `MAX_ENV_BUF_SZ` ?

Reply:
Yup, can do. I was being a bit silly about trying not to waste memory in this already kinda big struct, but a few hundred bytes on the stack is really not an issue.